readthedocs/rtd_tests/tests/test_footer.py

```diff
@@ -249,30 +249,36 @@ def render(self):
 
     def test_version_queries(self):
         # The number of Versions shouldn't impact the number of queries
-        with self.assertNumQueries(self.EXPECTED_QUERIES):
-            response = self.render()
-            self.assertContains(response, '0.8.1')
-
-        for patch in range(3):
-            identifier = '0.99.{}'.format(patch)
-            self.pip.versions.create(
-                verbose_name=identifier,
-                identifier=identifier,
-                type=TAG,
-                active=True,
-            )
-
-        with self.assertNumQueries(self.EXPECTED_QUERIES):
-            response = self.render()
-            self.assertContains(response, '0.99.0')
+        with mock.patch('readthedocs.api.v2.views.footer_views.increase_page_view_count') as mocked:
+            mocked.side_effect = None
+
+            with self.assertNumQueries(self.EXPECTED_QUERIES):
+                response = self.render()
+                self.assertContains(response, '0.8.1')
+
+            for patch in range(3):
+                identifier = '0.99.{}'.format(patch)
+                self.pip.versions.create(
+                    verbose_name=identifier,
+                    identifier=identifier,
+                    type=TAG,
+                    active=True,
+                )
+
+            with self.assertNumQueries(self.EXPECTED_QUERIES):
+                response = self.render()
+                self.assertContains(response, '0.99.0')
 
     def test_domain_queries(self):
         # Setting up a custom domain shouldn't impact the number of queries
-        self.pip.domains.create(
-            domain='http://docs.foobar.com',
-            canonical=True,
-        )
+        with mock.patch('readthedocs.api.v2.views.footer_views.increase_page_view_count') as mocked:
```

Comment:
Why do we need these changes?

Reply:
I'm guessing this is because otherwise that celery task will be called (with `CELERY_ALWAYS_EAGER`) and increase the number of queries.